crates/bevy_ecs/src/schedule/condition.rs

```diff
@@ -804,13 +809,11 @@ pub mod common_conditions {
     /// app.run(&mut world);
     /// assert_eq!(world.resource::<Counter>().0, 0);
     /// ```
+    #[deprecated]
```

Comment:
You should add the version it was deprecated in here. Makes it easier to delete later.

Reply:
should I note that as 0.13 then? (mainly since I don't know when this will get merged)

Reply:
that's fine. If it misses 0.13 it'll be easy to update.